Object Store/func.py

```diff
@@ -1,67 +1,85 @@
 import io
-import oci
-import re
 import os
+import gzip
 import json
-import sys
-import requests
 import logging
-import time
-import gzip
 
-from fdk import response
+import oci
+import requests
 
 
-def handler(ctx, data: io.BytesIO=None):
+def handler(ctx, data: io.BytesIO = None) -> None:
     try:
         body = json.loads(data.getvalue())
-        data = body.get("data", {})
-        additional_details = data.get("additionalDetails", {})
-        namespace = additional_details.get("namespace")
-        bucket = additional_details.get("bucketName")
-        obj = data.get("resourceName")
-        eventtime = body.get("eventTime")
+    except (Exception, ValueError) as ex:
+        logging.getLogger().info(str(ex))
```

Comment:
shouldn't it be error() instead of info()?

Reply:
Better yet, [logger.exception](https://docs.python.org/3/library/logging.html#logging.Logger.exception) will output a stack trace so there is no need to do `str(ex)`

Reply:
Overall agreed, was attempting to not change how logs were printed in this implementation. But can take a stab at updating and also add the logger at the module level.